server/base/src/main/java/org/apache/accumulo/server/util/MetadataTableUtil.java

```diff
@@ -894,12 +895,19 @@ public static void removeBulkLoadEntries(Connector conn, String tableId, long ti
         BatchWriter bw = conn.createBatchWriter(MetadataTable.NAME, new BatchWriterConfig())) {
       mscanner.setRange(new KeyExtent(tableId, null, null).toMetadataRange());
       mscanner.fetchColumnFamily(TabletsSection.BulkFileColumnFamily.NAME);
+      boolean shouldTrace = log.isTraceEnabled();
+      byte[] tidAsBytes = Long.toString(tid).getBytes();
```

Comment:
For predictability in different environments, should call `getBytes(StandardCharsets.UTF_8)`.  Value's toString method did something like this in the prev code.